### PR TITLE
Lindsay Cooper hmwk 1

### PR DESCRIPTION
**Pull request by lncooper**

### CODE REVIEW

File: hw01.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Lindsay Cooper 
+Hmwk 1
+
+I am a MAPSS student concentrating in psychology. I am especially interested in clinical and community psychology. I recently moved to Chicago from Boston. I am originally from Seattle. I was a preschool teacher for several years prior to joining MAPSS.
+
+My favorite spot in Chicago (so far):
+<em> https://www.artic.edu <em>
```
Comment (eugenehan98) [inline hw01.md:7]: I wasn't sure if you wanted to intentionally show the link but if you wanted to embed the link in a text, you can use the syntax [text](link url). What you used was in the cheat sheet that Ben handed out so really not a problem!

File: hw01.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+**Hobbies Include:**
```
Comment (eugenehan98) [inline hw01.md:9]: From what I'm seeing in the presentation format of the code, it seems like you were able to get both bold and italics.. I'm wondering how?

File: hw01.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+* hiking
+* painting 
+* reading
```
Comment (eugenehan98) [inline hw01.md:13]: Good use of bullet points!

File: hw01.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+
+![Diagnostics](https://cdn11.bigcommerce.com/s-jyvxk5hzsq/images/stencil/500x659/products/5355/43337/6077L__74971.1539347912.jpg?c=2&imbypass=on)
```
Comment (eugenehan98) [inline hw01.md:16]: Nice use of an image!

**Review by eugenehan98 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |    x       |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |     x      |              |            |

Remarks: I'm wondering how you were able to get the word "Hobbies" to have both bold and italics and your list of hobbies to be in italics without any italics notation/syntax. Otherwise, everything looks good!

File: hw01.md (hunk 2 above)
Comment (lncooper) [reply hw01.md:9]: Hi!

Thanks so much for your comments! 

I am not sure how/why that text remained bold and italics w/o accompanying code.  But, I think the code is something like this:

*__Hmwk 1__*

(combing the asterisks with the dashes) 

*italics* and _italics_
**bold** and __bold__ 

Source:
https://rstudio.com/wp-content/uploads/2015/02/rmarkdown-cheatsheet.pdf

Comment (lncooper) [reply hw01.md:9]: sorry! I just realized that it didn't print my code as code. 

But, you can use the asterisks for bold and the underscore for italics, or vice versa.

**Review by rrodriguezbarron — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |           |      x        |            |
| **Presentation**            |           |          x    |            |
| **Achievement, creativity** |           |              |    x        |
| **Ease of access**          |           |      x        |            |

Remarks:

* Lindsay, I couldn't see the image
* Good use of italics and lists
* The URL link is correct

**Pull request closed without merging**